src/experimental.rs

```diff
@@ -0,0 +1,18 @@
+//! Experimental methods.
+
+use super::*;
+
+pub mod native {
```

Comment:
I'd say this doesn't need to be exported.

Reply:
Yeah, you're right. Fixed